### PR TITLE
Feature/scope

### PR DESCRIPTION
**Pull request by s-block**

### CODE REVIEW

File: cla_backend/apps/checker/views.py [hunk 1]
```diff
@@ -134,3 +135,25 @@ def post_save(self, obj, created=False):
                 if form.is_valid():
                     form.save(obj.created_by)
                     notify_callback_created(obj)
+
+
+class DiagnosisViewSet(
+    PublicAPIViewSetMixin,
+    DiagnosisModelMixin,
+    mixins.CreateModelMixin,
+    mixins.RetrieveModelMixin,
```
Comment (r4vi) [inline cla_backend/apps/checker/views.py]: does this mean anyone can get back a diagnosis if they know the id? maybe we can lock it based on a session?

**Comment by r4vi** [comment]
another thought is do we want to keep around checker diagnosis objects lying around forever? maybe we can have a clear-up task

**Comment by s-block** [comment]
I was thinking about creating a celery task to delete them after a day or so. The diagnosis is never attached to the eligibility check so could be deleted after the contact form has been submitted

**Comment by r4vi** [comment]
yeah that doesn't need to be part of this PR anyway - more worried about someone brute forcing the guid space (small risk but we don't have any real ratelimit)

**Comment by r4vi** [comment]
:+1:

**Pull request merged**